src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final byte DATE_INDEX = 0;
+    private static final byte NAME_INDEX = 1;
+    private static final byte WORKING_DAYS_INDEX = 2;
+    private static final byte HOURLY_RATE_INDEX = 3;
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo);
+        // minus and plus one day because we need theese days inclusively,
+        // and I think it's better than to use two more .equals() in if() statement in a doubleloop
```

Comment:
remove these comments + if you don't agree with my comments write to the specific chats